README.md

```diff
@@ -29,14 +29,14 @@ $ npm install exports-loader --save-dev
 Then add the loader to the desired `require` calls. For example:
 
 ```js
-require('exports-loader?file,parse=helpers.parse!./file.js');
+require('exports-loader?run,parse=helpers.parse!./file.js');
```

Comment:
I'm not a maintainer here, but I'd recommend that for consistency you also update [the `src/globals.js` in the webpack docs related to Shimming > Global Exports](https://github.com/webpack/webpack.js.org/blob/master/src/content/guides/shimming.md#global-exports).